examples/tornado/app.py

```diff
@@ -0,0 +1,47 @@
+from asyncio import Queue
```

Comment:
This is the bigger problem than pathlib for py2 support and would need figuring out...

Reply:
Oh, true... maybe https://github.com/tornadoweb/tornado/blob/master/tornado/queues.py?

Reply:
I wouldn't worry about py2 support. We'll just call this a py3-only supported backend :)